test/test-kubernetes.js

```diff
@@ -168,8 +168,8 @@ describe('cloud-assets:kubernetes', function () {
 				console.log(JSON.stringify(serviceyml))
 
 				if (language === 'JAVA') {
-					assertYmlContent(serviceyml.spec.ports[1].name, 'http', 'serviceyml.spec.ports[0].name');
-					assertYmlContent(serviceyml.spec.ports[0].name, 'https', 'serviceyml.spec.ports[1].name');
+					assertYmlContent(serviceyml.spec.ports[0].name, 'https', 'serviceyml.spec.ports[0].name');
```

Comment:
cant tell if this is that change I made before showing up again or if you changed this back

Reply:
I just changed it back so it's 0 and then 1.